.gitignore

```diff
@@ -37,3 +37,6 @@ yarn-error.log*
 next-env.d.ts
 .vscode/settings.json
 .env
+
+# intellij
+.idea
```

Comment:
I use IntelliJ for DB connections sometimes, but don't want to commit project files.